templates/main/group_overview.html.ep

```diff
@@ -53,3 +62,27 @@
         %= include 'comments/add_comment_form_groups'
     % end
 % }
+
+<div class="panel panel-default filter-panel-bottom" id="filter-panel">
+    <div class="panel-heading"><strong>Filter</strong> <span>no filter present, click to toggle filter form</span></div>
+    <div class="panel-body">
+        <form action="#" method="get" id="filter-form">
+            <div class="form-group">
+                <strong>Misc</strong>
+                <input value="1" name="show_comments" type="checkbox" id="filter-show-comments">
+                <label for="filter-show-comments">Show comments</label>
+                <%= help_popover('Help for <i>Show comments</i>' => '<p>Show comments for current group overview</p>')
+                %>
+                <input value="1" name="latest_comment" type="checkbox" id="filter-latest-comment">
+                <label for="filter-latest-comment">Only latest comment</label>
+                <%= help_popover('Help for <i>Only latest comment</i>' => '<p>Show only the latest comment</p>')
+                %>
+                <input value="1" name="fullscreen" type="checkbox" id="filter-fullscreen">
+                <label for="filter-fullscreen">Full screen</label>
+                <%= help_popover('Help for <i>Full screen</i>' => '<p>Show group overview in full screen mode</p>')
+                %>
+            </div>
+            <button type="submit" class="btn btn-default">Apply</button>
+        </form>
+    </div>
+</div>
```

Comment:
I suggest to focus on one thing at a time in this PR, e.g. either full screen or "show comments" but the general idea is very much appreciated. But keep in mind that we have now a "limit to …" line just below the build bars and this dynamic filter box further below the comments. Probably one of two approaches should be followed consistently.